packages/stylist-core/src/ast/rule_block_content.rs

```diff
@@ -0,0 +1,24 @@
+use super::{Block, Rule, StyleAttribute, StyleContext, ToStyleStr};
+
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+pub enum RuleBlockContent {
```

Comment:
I think unless we change `Block` to `QualifiedRule`, then it will be somewhat confusing to change `RuleBlockContent` to `BlockContent`.

Personally, I think we should revisit naming along with or after #40.